lib/linux_admin/service/sys_v_init_service.rb

```diff
@@ -42,5 +42,14 @@ def restart
 
       self
     end
+
+    def reload
```

Comment:
They are close, but not exact copies. Actually they have the same differences as all the other methods.